core/sparqlbuilder/src/test/java/org/eclipse/rdf4j/sparqlbuilder/examples/updatespec/Section3Test.java

```diff
@@ -342,4 +342,30 @@ public void example_14() {
 	public void example_15() {
 		p(Queries.ADD().fromDefault().to(iri("http://example.org/named")));
 	}
+
+	/**
+	 * DELETE { GRAPH ?g1 { ?subject <http://my-example.com/anyIRI/> ?object . ?subject ?predicate
+	 * <http://my-example.com/anyIRI/> . } } WHERE { GRAPH ?g1 { OPTIONAL { ?subject ?predicate
+	 * <http://my-example.com/anyIRI/> . } OPTIONAL { ?subject <http://my-example.com/anyIRI/> ?object . } } }
+	 */
+	@Test
+	public void example_issue_1481() {
+		ModifyQuery modify = Queries.MODIFY();
+		Iri g1 = () -> "<g1>";
+
+		Variable subject = SparqlBuilder.var("subject");
+		Variable obj = SparqlBuilder.var("object");
+		Prefix pre = SparqlBuilder.prefix(iri("http://my-example.com/anyIRI/"));
```

Comment:
Could you use full variable names? I know we haven't been so great at doing it in the code base in general.